cobalt/h5vcc/h5vcc_settings.idl

```diff
@@ -14,4 +14,5 @@
 
 interface H5vccSettings {
   boolean set(DOMString name, (long or DOMString) value);
+  DOMString getPersistentSetting(DOMString name, DOMString default_setting);
```

Comment:
Basically to add a symmetric setter alongside this getter? That sounds good to me. Why int over string, is it because it's simpler and the persistent setting values can (for now) all be captured as ints?

Reply:
The concern with simple pass-throughs is that the setting name is different. Currently with `set()` we use `QUIC` for example which is then translated to `kQuicEnabledPersistentSettingsKey` / `QUICEnabled`.

Also with the simple pass-throughs quic for example uses `getPersistentSettingAsBool` which doesn't provide the ability to distinguish from a failed persistent value get from a default value.

Reply:
updated!